pkgs/development/mobile/androidenv/androidndk.nix

```diff
@@ -33,9 +33,14 @@ stdenv.mkDerivation rec {
   in ''
     set -x
     mkdir -pv $out/libexec
+    mkdir -pv $out/lib64
+    ln -s ${ncurses.out}/lib/libncursesw.so.6 $out/lib64/libtinfo.so.5
+    ln -s ${ncurses.out}/lib/libncurses.so.6 $out/lib64/libncurses.so.5
```

Comment:
Erm this doesn't seem like a good idea.  Grab these from `ncurses5` instead?

Reply:
Fixed